{"url": "https://api.github.com/repos/pytorch/pytorch/issues/comments/333324826", "html_url": "https://github.com/pytorch/pytorch/issues/494#issuecomment-333324826", "issue_url": "https://api.github.com/repos/pytorch/pytorch/issues/494", "id": 333324826, "node_id": "MDEyOklzc3VlQ29tbWVudDMzMzMyNDgyNg==", "user": {"login": "QuantScientist", "id": 18743986, "node_id": "MDQ6VXNlcjE4NzQzOTg2", "avatar_url": "https://avatars2.githubusercontent.com/u/18743986?v=4", "gravatar_id": "", "url": "https://api.github.com/users/QuantScientist", "html_url": "https://github.com/QuantScientist", "followers_url": "https://api.github.com/users/QuantScientist/followers", "following_url": "https://api.github.com/users/QuantScientist/following{/other_user}", "gists_url": "https://api.github.com/users/QuantScientist/gists{/gist_id}", "starred_url": "https://api.github.com/users/QuantScientist/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/QuantScientist/subscriptions", "organizations_url": "https://api.github.com/users/QuantScientist/orgs", "repos_url": "https://api.github.com/users/QuantScientist/repos", "events_url": "https://api.github.com/users/QuantScientist/events{/privacy}", "received_events_url": "https://api.github.com/users/QuantScientist/received_events", "type": "User", "site_admin": false}, "created_at": "2017-09-30T17:51:16Z", "updated_at": "2017-09-30T17:51:16Z", "author_association": "NONE", "body_html": "<p><a class=\"user-mention\" data-hovercard-type=\"user\" data-hovercard-url=\"/hovercards?user_id=9998726\" data-octo-click=\"hovercard-link-click\" data-octo-dimensions=\"link_type:self\" href=\"https://github.com/peterjc123\">@peterjc123</a><br>\nNew problem ...<br>\n````RuntimeError: CUDNN_STATUS_NOT_SUPPORTED. This error may appear if you passed in a non-contiguous input.```</p>\n<p>Works great on Linux, exact same code.<br>\n<a href=\"https://github.com/QuantScientist/Deep-Learning-Boot-Camp/blob/master/day02-PyTORCH-and-PyCUDA/PyTorch/18-PyTorch-NUMER.AI-Binary-Classification-BCELoss-0.691839667509%20.ipynb\">https://github.com/QuantScientist/Deep-Learning-Boot-Camp/blob/master/day02-PyTORCH-and-PyCUDA/PyTorch/18-PyTorch-NUMER.AI-Binary-Classification-BCELoss-0.691839667509%20.ipynb</a></p>\n<p><a class=\"issue-link js-issue-link\" data-error-text=\"Failed to load issue title\" data-id=\"213195830\" data-permission-text=\"Issue title is private\" data-url=\"https://github.com/pytorch/pytorch/issues/967\" data-hovercard-type=\"issue\" data-hovercard-url=\"/pytorch/pytorch/issues/967/hovercard\" href=\"https://github.com/pytorch/pytorch/issues/967\">#967</a><br>\n<a href=\"https://discuss.pytorch.org/t/resolved-batchnorm1d-cudnn-status-not-supported/3049/7\" rel=\"nofollow\">https://discuss.pytorch.org/t/resolved-batchnorm1d-cudnn-status-not-supported/3049/7</a></p>\n<p>I have CUDA 8 and CuDNN 6, which versions did you use?</p>", "body_text": "@peterjc123\nNew problem ...\n````RuntimeError: CUDNN_STATUS_NOT_SUPPORTED. This error may appear if you passed in a non-contiguous input.```\nWorks great on Linux, exact same code.\nhttps://github.com/QuantScientist/Deep-Learning-Boot-Camp/blob/master/day02-PyTORCH-and-PyCUDA/PyTorch/18-PyTorch-NUMER.AI-Binary-Classification-BCELoss-0.691839667509%20.ipynb\n#967\nhttps://discuss.pytorch.org/t/resolved-batchnorm1d-cudnn-status-not-supported/3049/7\nI have CUDA 8 and CuDNN 6, which versions did you use?", "body": "@peterjc123 \r\nNew problem ...\r\n````RuntimeError: CUDNN_STATUS_NOT_SUPPORTED. This error may appear if you passed in a non-contiguous input.```\r\n\r\nWorks great on Linux, exact same code. \r\nhttps://github.com/QuantScientist/Deep-Learning-Boot-Camp/blob/master/day02-PyTORCH-and-PyCUDA/PyTorch/18-PyTorch-NUMER.AI-Binary-Classification-BCELoss-0.691839667509%20.ipynb\r\n\r\nhttps://github.com/pytorch/pytorch/issues/967\r\nhttps://discuss.pytorch.org/t/resolved-batchnorm1d-cudnn-status-not-supported/3049/7\r\n\r\nI have CUDA 8 and CuDNN 6, which versions did you use? \r\n"}